defaults/main.yml

```diff
@@ -161,6 +164,41 @@ docker__listen: [ '{{ docker__tcp_listen }}' ]
 docker__labels: {}
 
                                                                    # ]]]
+# .. envvar:: docker__debug [[[
+#
+# Start docker daemon in debug mode.
+#
+# .. code-block:: yaml
+#    :linenos:
+#
+#    docker__debug: true
```

Comment:
You don't need to provide examples like this for each and every variable, they don't add anything important. Variables a few choices could be presented as a simple list, more complicated variables should go into `defaults-detailed.rst` file. But for the Boolean variables like this one you don't need to provide an example `code-block`, since the configuration of a boolean should be obvious.